editor.planx.uk/src/@planx/components/Feedback/Public/Public.tsx

```diff
@@ -45,18 +43,23 @@ const FeedbackComponent = (
   return (
     <Card handleSubmit={formik.handleSubmit}>
       <CardHeader title={props.title} />
-      <Typography pt={4}>
-        This service is a work in progress, any feedback you share about your
-        experience will help us to improve it.
-      </Typography>
-      <Typography pt={2}>
-        Don't share any personal or financial information in your feedback. If
-        you do we will act according to our{" "}
-        <Link href={props.privacyPolicyLink ?? ""}>privacy policy</Link>.
```

Comment:
It's now up to the user to put a privacy policy link in the description via RichTextInput if they want to.